src/js/user_activity.js

```diff
@@ -0,0 +1,55 @@
+var interval = 0;
```

Comment:
I know naming is a bit all over the place in the theme, but this script does more than monitoring user activity, it also takes action (hiding / showing the toggles). Please rename it to "side-content-toggles.js" (you can use `git mv` for that)